frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -284,4 +255,42 @@ function BookInfo:onShowBookCover(file)
     end
 end
 
+function BookInfo:getCurrentPageLineWordCounts()
+    if self.ui.rolling then
+        local Screen = require("device").screen
+        local res = self.ui.document._document:getTextFromPositions(0, 0, Screen:getWidth(), Screen:getHeight(),
+            false, false) -- do not highlight
+        if not res then return end
+        local lines = self.ui.document:getScreenBoxesFromPositions(res.pos0, res.pos1, true)
+        local words = {}
+        for word in util.gsplit(res.text, "[%s%p]+", false) do
+            if util.hasCJKChar(word) then
+                for char in util.gsplit(word, "[\192-\255][\128-\191]+", true) do
+                    table.insert(words, char)
+                end
+            else
+                table.insert(words, word)
```

Comment:
You could just increment a counter, instead of accumulating in the table `words`.